### PR TITLE
Add test workflow for TestApp

### PR DESCRIPTION
**Pull request by ctp27**
- Added back test workflow

### CODE REVIEW

File: .github/workflows/translations_workflow.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+jobs:
+  formatCode:
+    name: Translate Strings
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: '0'
+      - name: translate strings
+        uses: ./
```
Comment (ctp27) [inline .github/workflows/translations_workflow.yml:12]: @ndahlquist Is this okay or should we reference it using the release?

Comment (ndahlquist) [reply .github/workflows/translations_workflow.yml:12]: I think this is good, if we do releases we'll need to change it every time.

File: lib.sh [hunk 2]
```diff
@@ -5,7 +5,7 @@ _git_is_dirty() {
 }
 
 _commit_and_push() {
-	git add app/src/main/res/*
+	git add .
```
Comment (ndahlquist) [inline lib.sh:8]: good catch

File: .github/workflows/translations_workflow.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+        with:
+          translationKey: ${{ secrets.GOOGLE_TRANSLATE_API_KEY }}
+          outputLanguages: 'es,fr,zh'
+          resDirectories: 'test_app/app/src/main/res/'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (ndahlquist) [inline .github/workflows/translations_workflow.yml:18]: This is actually not necessary (I think it was used by the old lib.sh, but was removed)

**Pull request merged**